### ISSUE TITLE
Beomote as sender

### ISSUE THREAD

**Issue by Ion3** [christianlykke9/Beomote#1]
Hej,

is it possible to use your library for Arduino to control B&O devices via IR? I understand your example to receive commands from the BEO4, but do you also have an example where the Arduino controls B&O devices via IR? (e.g. turning on TV by pressing a button on the Arduino)

Mange tak,
Timo

**Comment by monin** [comment]
Hi, 
Does anybody still have the SendRepository for BEO4 accessible? It seems to be deleted from PolyVection github.
Thanks.
Daniel

**Comment by alow2004** [comment]
Hi Philip, i have the same problem as Daniel,can you please share your work with us? , the polyvection folder is not available to us anymore. For my part, i try to interface alexa (amazon echo dot) and an arduino to pilot a beosound device...

**Comment by frank-trampe** [comment]
@PolyPv or @lon3, does one of you still have the PolyIR code as described above?

**Comment by cdf1982** [comment]
Yes, @PolyPv or @lon3, I'd also be very thankful if you have the PolyIR code described above... I'm looking for a solution to control a B&O TV (quite old, it's a BeoSystem 3) and it looks like you were the only one on the whole internet who got somewhere...

**Comment by Ion3** [comment]
Dear @cdf1982,
dear @frank-trampe, 

I hope the attached helps. I haven't progressed in this project. Do you think there might be a solution to implement the Bang & Olufsen IR Codes in one of this projects?

- [Tasmota YTF IR Bridge](https://github.com/arendst/Tasmota/wiki/ytf-ir-bridge)
- [ESP8266-HTTP-IR-Blaster](https://github.com/mdhiggins/ESP8266-HTTP-IR-Blaster)

Greetings,
Timo

[PolyIR-master.zip](https://github.com/christianlykke9/Beomote/files/4129123/PolyIR-master.zip)
